libsolidity/analysis/TypeChecker.cpp

```diff
@@ -3827,7 +3827,7 @@ void TypeChecker::endVisit(Literal const& _literal)
 
 			optional<string> parameterTypeMessage;
 			if (parameterCountMessage.has_value())
-				m_errorReporter.typeError(4778_error, _literal.location(), parameterCountMessage.value());
+				m_errorReporter.typeError(9128_error, _literal.location(), parameterCountMessage.value());
```

Comment:
Why a different number?

Reply:
This was the result of running `./scripts/error_codes.py --fix`, it identified that ID 4778 appears 2 times.

Reply:
Error 4778 was already present in [libyul/AsmAnalysis.cpp:L703](https://github.com/ethereum/solidity/blob/247df5509931d4ad747d2d926432a0955bb2e6f5/libyul/AsmAnalysis.cpp#L703).

Reply:
Oh, interesting. This number was actually in the initial implementation I took over from Chris but it does look like a genuine duplicate.